www/src/components/repos/packages/ChartReadme.js

```diff
@@ -0,0 +1,15 @@
+import { useContext } from 'react'
+
+import { Box, Markdown } from 'grommet'
+
+import ChartContext from '../../../contexts/ChartContext'
+
+import { MARKDOWN_STYLING } from '../Chart'
+
+export default function ChartReadme() {
+  const { current } = useContext(ChartContext)
+
+  return (
+    <Box pad="large"><Markdown components={MARKDOWN_STYLING}>{current?.readme || 'n/a'}</Markdown></Box>
```

Comment:
we can use react routers `Outlet` component to manage this a bit smoother like in other views

Reply:
[I've used it already in a way like it is done for repositories](https://github.com/pluralsh/plural/pull/376/files#diff-8ca897a150f2157542874fde88bd45e021a3f748f1f1a3d38ba6b0fdf88c761bR291). I'm not sure if that's enough or you would like to move the `Outlet` placement or add another one here for different purpose. Can you explain?

Reply:
Oh I see you are structuring it right, where is the Outlet component injected in the component tree?  I just couldn't find it in the diff anywhere

Reply:
It's in Plural.js if i understand you correctly. There are only two of them at the moment as it's still in progress.